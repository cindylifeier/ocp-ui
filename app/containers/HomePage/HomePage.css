.header {
  grid-area: header;
}

.content {
  grid-area: content;
}

.panel {
  grid-area: sidebar;
}

.gridContainer {
  color: #444;
  width: auto;
  background-color: #fff;
  margin: 0 auto;
  display: grid;
  grid-row-gap: 5px;
  grid-template-areas:
    "header"
    "content"
    "sidebar";
}

.gridItem {
  background-color: #fff;
  color: #fff;
  border-radius: 5px;
  padding: 3px;
  font-size: 150%;
}

/* Desktops and laptops ----------- */
@media only screen and (min-width: 1224px) {
  .gridContainer {
    grid-template-columns: 1fr 4fr;
    grid-template-areas:
      "header header"
      "sidebar content";
  }
}
