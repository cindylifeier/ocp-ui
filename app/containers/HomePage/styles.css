.gridContainer {
  color: #444;
  width: auto;
  background-color: #fff;
  margin: 0 auto;
  display: grid;
  grid-row-gap: 5px;
  grid-template-areas:
    "sidebar"
    "content";
}

.content {
  grid-area: content;
}

.panel {
  grid-area: sidebar;
}

.gridItem {
  background-color: #fff;
  color: rgb(51, 51, 51);
  border-radius: 5px;
  padding: 3px;
  font-size: 150%;
}

/* Desktops and laptops ----------- */
@media only screen and (min-width: 1224px) {
  .gridContainer {
    grid-template-columns: 1fr 4fr;
    grid-template-areas:
      "sidebar content";
  }
}
