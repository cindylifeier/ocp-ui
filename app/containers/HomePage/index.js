/*
 * HomePage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 * NOTE: while this component should technically be a stateless functional
 * component (SFC), hot reloading does not currently support SFCs. If hot
 * reloading is not a necessity for you then you can refactor it and remove
 * the linting exception.
 */

import React from 'react';
import { Helmet } from 'react-helmet';
// import { FormattedMessage } from 'react-intl';
import GoldenLayout from '../../components/GoldenLayout/Loadable';
import renderOrganizations from '../Organizations/render';
import renderPractitioners from '../Practitioners/render';
import renderLocationsComponent from '../../containers/Locations/render';
import renderHealthcareServicesComponent from '../HealthcareServices/render';
import SideBar from '../../components/SideBar';
import styles from './styles.css';

// import messages from './messages';

const initialStateMetadata =
  {
    settings: {
      hasHeaders: true,
      constrainDragToContainer: false,
      reorderEnabled: true,
      selectionEnabled: false,
      popoutWholeStack: false,
      blockedPopoutsThrowError: true,
      closePopoutsOnUnload: true,
      showPopoutIcon: false,
      showMaximiseIcon: true,
      showCloseIcon: true,
      responsiveMode: 'onload',
      tabOverlapAllowance: 0,
      reorderOnTabMenuClick: true,
      tabControlOffset: 10,
    },
    dimensions: {
      borderWidth: 5,
      borderGrabWidth: 15,
      minItemHeight: 10,
      minItemWidth: 10,
      headerHeight: 20,
      dragProxyWidth: 300,
      dragProxyHeight: 200,
    },
    labels: {
      close: 'close',
      maximise: 'maximise',
      minimise: 'minimise',
      popout: 'open in new window',
      popin: 'pop in',
      tabDropdown: 'additional tabs',
    },
    content: [{
      type: 'row',
      isClosable: true,
      reorderEnabled: true,
      title: '',
      content: [{
        type: 'column',
        isClosable: true,
        reorderEnabled: true,
        title: '',
        width: 50,
        content: [{
          type: 'stack',
          header: {},
          isClosable: true,
          reorderEnabled: true,
          title: '',
          activeItemIndex: 0,
          height: 50,
          content: [{
            title: 'Organizations',
            type: 'component',
            componentName: 'organizations',
            isClosable: true,
            reorderEnabled: true,
          },
          ],
        }, {
          type: 'stack',
          header: {},
          isClosable: true,
          reorderEnabled: true,
          title: '',
          activeItemIndex: 0,
          width: 50,
          height: 50,
          content: [{
            title: 'Practitioners',
            type: 'component',
            componentName: 'practitioners',
            isClosable: true,
            reorderEnabled: true,
          },
          ],
        },
        ],
      }, {
        type: 'column',
        isClosable: true,
        reorderEnabled: true,
        title: '',
        width: 50,
        content: [{
          type: 'stack',
          header: {},
          isClosable: true,
          reorderEnabled: true,
          title: '',
          activeItemIndex: 0,
          height: 50,
          content: [{
            title: 'Location',
            type: 'component',
            componentName: 'locations',
            isClosable: true,
            reorderEnabled: true,
          },
          ],
        }, {
          type: 'stack',
          width: 50,
          height: 50,
          isClosable: true,
          reorderEnabled: true,
          title: '',
          activeItemIndex: 0,
          content: [{
            title: 'Healthcare Services',
            type: 'component',
            componentName: 'healthcareServices',
            isClosable: true,
            reorderEnabled: true,
          },
          ],
        },
        ],
      },
      ],
    },
    ],
    isClosable: true,
    reorderEnabled: true,
    title: '',
    openPopouts: [],
    maximisedItemId: null,
  };

const componentMetadata = [
  { name: 'organizations', text: 'Organizations', factoryMethod: renderOrganizations },
  { name: 'practitioners', text: 'Practitioners', factoryMethod: renderPractitioners },
  { name: 'locations', text: 'Locations', factoryMethod: renderLocationsComponent },
  { name: 'healthcareServices', text: 'Healthcare Services', factoryMethod: renderHealthcareServicesComponent },
];

export default class HomePage extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
  render() {
    return (
      <div>
        <Helmet>
          <title>Home</title>
          <meta name="description" content="home page of Omnibus Care Plan application" />
        </Helmet>
        <div className={styles.gridContainer}>
          <div className={`${styles.gridItem} ${styles.panel}`}>
            <SideBar />
          </div>
          <div className={`${styles.gridItem} ${styles.content}`}>
            <GoldenLayout
              containerId="golden-home"
              componentMetadata={componentMetadata}
              stateMetadata={initialStateMetadata}
            />
          </div>
        </div>
      </div>
    );
  }
}
