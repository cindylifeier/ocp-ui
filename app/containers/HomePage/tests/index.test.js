import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import 'mock-local-storage';

import HomePage from '../index';

configure({ adapter: new Adapter() });

describe('<HomePage />', () => {
  describe('snapshot tests', () => {
    it('should render the page message', () => {
      // Act
      const renderedComponent = shallow(
        <HomePage />,
      );

      // Assert
      expect(renderedComponent).toMatchSnapshot();
    });
  });
});
