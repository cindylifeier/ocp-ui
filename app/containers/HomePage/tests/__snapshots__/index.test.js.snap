// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> snapshot tests should render the page message 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HomePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          encodeSpecialCharacters={true}
        >
          <title>
            Home
          </title>
          <meta
            content="home page of Omnibus Care Plan application"
            name="description"
          />
        </HelmetWrapper>,
        <LoadableComponent
          componentMetadata={
            Array [
              Object {
                "factoryMethod": [Function],
                "name": "organizations",
                "text": "Organizations",
              },
              Object {
                "factoryMethod": [Function],
                "name": "practitioners",
                "text": "Practitioners",
              },
              Object {
                "factoryMethod": [Function],
                "name": "locations",
                "text": "Locations",
              },
              Object {
                "factoryMethod": [Function],
                "name": "healthcareServices",
                "text": "Healthcare Services",
              },
            ]
          }
          containerId="golden-home"
          stateMetadata={
            Object {
              "content": Array [
                Object {
                  "content": Array [
                    Object {
                      "content": Array [
                        Object {
                          "activeItemIndex": 0,
                          "content": Array [
                            Object {
                              "componentName": "organizations",
                              "isClosable": true,
                              "reorderEnabled": true,
                              "title": "Organizations",
                              "type": "component",
                            },
                          ],
                          "header": Object {},
                          "height": 50,
                          "isClosable": true,
                          "reorderEnabled": true,
                          "title": "",
                          "type": "stack",
                        },
                        Object {
                          "activeItemIndex": 0,
                          "content": Array [
                            Object {
                              "componentName": "practitioners",
                              "isClosable": true,
                              "reorderEnabled": true,
                              "title": "Practitioners",
                              "type": "component",
                            },
                          ],
                          "header": Object {},
                          "height": 50,
                          "isClosable": true,
                          "reorderEnabled": true,
                          "title": "",
                          "type": "stack",
                          "width": 50,
                        },
                      ],
                      "isClosable": true,
                      "reorderEnabled": true,
                      "title": "",
                      "type": "column",
                      "width": 50,
                    },
                    Object {
                      "content": Array [
                        Object {
                          "activeItemIndex": 0,
                          "content": Array [
                            Object {
                              "componentName": "locations",
                              "isClosable": true,
                              "reorderEnabled": true,
                              "title": "Location",
                              "type": "component",
                            },
                          ],
                          "header": Object {},
                          "height": 50,
                          "isClosable": true,
                          "reorderEnabled": true,
                          "title": "",
                          "type": "stack",
                        },
                        Object {
                          "activeItemIndex": 0,
                          "content": Array [
                            Object {
                              "componentName": "healthcareServices",
                              "isClosable": true,
                              "reorderEnabled": true,
                              "title": "Healthcare Services",
                              "type": "component",
                            },
                          ],
                          "height": 50,
                          "isClosable": true,
                          "reorderEnabled": true,
                          "title": "",
                          "type": "stack",
                          "width": 50,
                        },
                      ],
                      "isClosable": true,
                      "reorderEnabled": true,
                      "title": "",
                      "type": "column",
                      "width": 50,
                    },
                  ],
                  "isClosable": true,
                  "reorderEnabled": true,
                  "title": "",
                  "type": "row",
                },
              ],
              "dimensions": Object {
                "borderGrabWidth": 15,
                "borderWidth": 5,
                "dragProxyHeight": 200,
                "dragProxyWidth": 300,
                "headerHeight": 30,
                "minItemHeight": 10,
                "minItemWidth": 10,
              },
              "isClosable": true,
              "labels": Object {
                "close": "close",
                "maximise": "maximise",
                "minimise": "minimise",
                "popin": "pop in",
                "popout": "open in new window",
                "tabDropdown": "additional tabs",
              },
              "maximisedItemId": null,
              "openPopouts": Array [],
              "reorderEnabled": true,
              "settings": Object {
                "blockedPopoutsThrowError": true,
                "closePopoutsOnUnload": true,
                "constrainDragToContainer": false,
                "hasHeaders": true,
                "popoutWholeStack": false,
                "reorderEnabled": true,
                "reorderOnTabMenuClick": true,
                "responsiveMode": "onload",
                "selectionEnabled": false,
                "showCloseIcon": true,
                "showMaximiseIcon": true,
                "showPopoutIcon": false,
                "tabControlOffset": 10,
                "tabOverlapAllowance": 0,
              },
              "title": "",
            }
          }
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <title>
              Home
            </title>,
            <meta
              content="home page of Omnibus Care Plan application"
              name="description"
            />,
          ],
          "encodeSpecialCharacters": true,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Home",
            },
            "ref": null,
            "rendered": "Home",
            "type": "title",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "home page of Omnibus Care Plan application",
              "name": "description",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "componentMetadata": Array [
            Object {
              "factoryMethod": [Function],
              "name": "organizations",
              "text": "Organizations",
            },
            Object {
              "factoryMethod": [Function],
              "name": "practitioners",
              "text": "Practitioners",
            },
            Object {
              "factoryMethod": [Function],
              "name": "locations",
              "text": "Locations",
            },
            Object {
              "factoryMethod": [Function],
              "name": "healthcareServices",
              "text": "Healthcare Services",
            },
          ],
          "containerId": "golden-home",
          "stateMetadata": Object {
            "content": Array [
              Object {
                "content": Array [
                  Object {
                    "content": Array [
                      Object {
                        "activeItemIndex": 0,
                        "content": Array [
                          Object {
                            "componentName": "organizations",
                            "isClosable": true,
                            "reorderEnabled": true,
                            "title": "Organizations",
                            "type": "component",
                          },
                        ],
                        "header": Object {},
                        "height": 50,
                        "isClosable": true,
                        "reorderEnabled": true,
                        "title": "",
                        "type": "stack",
                      },
                      Object {
                        "activeItemIndex": 0,
                        "content": Array [
                          Object {
                            "componentName": "practitioners",
                            "isClosable": true,
                            "reorderEnabled": true,
                            "title": "Practitioners",
                            "type": "component",
                          },
                        ],
                        "header": Object {},
                        "height": 50,
                        "isClosable": true,
                        "reorderEnabled": true,
                        "title": "",
                        "type": "stack",
                        "width": 50,
                      },
                    ],
                    "isClosable": true,
                    "reorderEnabled": true,
                    "title": "",
                    "type": "column",
                    "width": 50,
                  },
                  Object {
                    "content": Array [
                      Object {
                        "activeItemIndex": 0,
                        "content": Array [
                          Object {
                            "componentName": "locations",
                            "isClosable": true,
                            "reorderEnabled": true,
                            "title": "Location",
                            "type": "component",
                          },
                        ],
                        "header": Object {},
                        "height": 50,
                        "isClosable": true,
                        "reorderEnabled": true,
                        "title": "",
                        "type": "stack",
                      },
                      Object {
                        "activeItemIndex": 0,
                        "content": Array [
                          Object {
                            "componentName": "healthcareServices",
                            "isClosable": true,
                            "reorderEnabled": true,
                            "title": "Healthcare Services",
                            "type": "component",
                          },
                        ],
                        "height": 50,
                        "isClosable": true,
                        "reorderEnabled": true,
                        "title": "",
                        "type": "stack",
                        "width": 50,
                      },
                    ],
                    "isClosable": true,
                    "reorderEnabled": true,
                    "title": "",
                    "type": "column",
                    "width": 50,
                  },
                ],
                "isClosable": true,
                "reorderEnabled": true,
                "title": "",
                "type": "row",
              },
            ],
            "dimensions": Object {
              "borderGrabWidth": 15,
              "borderWidth": 5,
              "dragProxyHeight": 200,
              "dragProxyWidth": 300,
              "headerHeight": 30,
              "minItemHeight": 10,
              "minItemWidth": 10,
            },
            "isClosable": true,
            "labels": Object {
              "close": "close",
              "maximise": "maximise",
              "minimise": "minimise",
              "popin": "pop in",
              "popout": "open in new window",
              "tabDropdown": "additional tabs",
            },
            "maximisedItemId": null,
            "openPopouts": Array [],
            "reorderEnabled": true,
            "settings": Object {
              "blockedPopoutsThrowError": true,
              "closePopoutsOnUnload": true,
              "constrainDragToContainer": false,
              "hasHeaders": true,
              "popoutWholeStack": false,
              "reorderEnabled": true,
              "reorderOnTabMenuClick": true,
              "responsiveMode": "onload",
              "selectionEnabled": false,
              "showCloseIcon": true,
              "showMaximiseIcon": true,
              "showPopoutIcon": false,
              "tabControlOffset": 10,
              "tabOverlapAllowance": 0,
            },
            "title": "",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            encodeSpecialCharacters={true}
          >
            <title>
              Home
            </title>
            <meta
              content="home page of Omnibus Care Plan application"
              name="description"
            />
          </HelmetWrapper>,
          <LoadableComponent
            componentMetadata={
              Array [
                Object {
                  "factoryMethod": [Function],
                  "name": "organizations",
                  "text": "Organizations",
                },
                Object {
                  "factoryMethod": [Function],
                  "name": "practitioners",
                  "text": "Practitioners",
                },
                Object {
                  "factoryMethod": [Function],
                  "name": "locations",
                  "text": "Locations",
                },
                Object {
                  "factoryMethod": [Function],
                  "name": "healthcareServices",
                  "text": "Healthcare Services",
                },
              ]
            }
            containerId="golden-home"
            stateMetadata={
              Object {
                "content": Array [
                  Object {
                    "content": Array [
                      Object {
                        "content": Array [
                          Object {
                            "activeItemIndex": 0,
                            "content": Array [
                              Object {
                                "componentName": "organizations",
                                "isClosable": true,
                                "reorderEnabled": true,
                                "title": "Organizations",
                                "type": "component",
                              },
                            ],
                            "header": Object {},
                            "height": 50,
                            "isClosable": true,
                            "reorderEnabled": true,
                            "title": "",
                            "type": "stack",
                          },
                          Object {
                            "activeItemIndex": 0,
                            "content": Array [
                              Object {
                                "componentName": "practitioners",
                                "isClosable": true,
                                "reorderEnabled": true,
                                "title": "Practitioners",
                                "type": "component",
                              },
                            ],
                            "header": Object {},
                            "height": 50,
                            "isClosable": true,
                            "reorderEnabled": true,
                            "title": "",
                            "type": "stack",
                            "width": 50,
                          },
                        ],
                        "isClosable": true,
                        "reorderEnabled": true,
                        "title": "",
                        "type": "column",
                        "width": 50,
                      },
                      Object {
                        "content": Array [
                          Object {
                            "activeItemIndex": 0,
                            "content": Array [
                              Object {
                                "componentName": "locations",
                                "isClosable": true,
                                "reorderEnabled": true,
                                "title": "Location",
                                "type": "component",
                              },
                            ],
                            "header": Object {},
                            "height": 50,
                            "isClosable": true,
                            "reorderEnabled": true,
                            "title": "",
                            "type": "stack",
                          },
                          Object {
                            "activeItemIndex": 0,
                            "content": Array [
                              Object {
                                "componentName": "healthcareServices",
                                "isClosable": true,
                                "reorderEnabled": true,
                                "title": "Healthcare Services",
                                "type": "component",
                              },
                            ],
                            "height": 50,
                            "isClosable": true,
                            "reorderEnabled": true,
                            "title": "",
                            "type": "stack",
                            "width": 50,
                          },
                        ],
                        "isClosable": true,
                        "reorderEnabled": true,
                        "title": "",
                        "type": "column",
                        "width": 50,
                      },
                    ],
                    "isClosable": true,
                    "reorderEnabled": true,
                    "title": "",
                    "type": "row",
                  },
                ],
                "dimensions": Object {
                  "borderGrabWidth": 15,
                  "borderWidth": 5,
                  "dragProxyHeight": 200,
                  "dragProxyWidth": 300,
                  "headerHeight": 30,
                  "minItemHeight": 10,
                  "minItemWidth": 10,
                },
                "isClosable": true,
                "labels": Object {
                  "close": "close",
                  "maximise": "maximise",
                  "minimise": "minimise",
                  "popin": "pop in",
                  "popout": "open in new window",
                  "tabDropdown": "additional tabs",
                },
                "maximisedItemId": null,
                "openPopouts": Array [],
                "reorderEnabled": true,
                "settings": Object {
                  "blockedPopoutsThrowError": true,
                  "closePopoutsOnUnload": true,
                  "constrainDragToContainer": false,
                  "hasHeaders": true,
                  "popoutWholeStack": false,
                  "reorderEnabled": true,
                  "reorderOnTabMenuClick": true,
                  "responsiveMode": "onload",
                  "selectionEnabled": false,
                  "showCloseIcon": true,
                  "showMaximiseIcon": true,
                  "showPopoutIcon": false,
                  "tabControlOffset": 10,
                  "tabOverlapAllowance": 0,
                },
                "title": "",
              }
            }
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <title>
                Home
              </title>,
              <meta
                content="home page of Omnibus Care Plan application"
                name="description"
              />,
            ],
            "encodeSpecialCharacters": true,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Home",
              },
              "ref": null,
              "rendered": "Home",
              "type": "title",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "content": "home page of Omnibus Care Plan application",
                "name": "description",
              },
              "ref": null,
              "rendered": null,
              "type": "meta",
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "componentMetadata": Array [
              Object {
                "factoryMethod": [Function],
                "name": "organizations",
                "text": "Organizations",
              },
              Object {
                "factoryMethod": [Function],
                "name": "practitioners",
                "text": "Practitioners",
              },
              Object {
                "factoryMethod": [Function],
                "name": "locations",
                "text": "Locations",
              },
              Object {
                "factoryMethod": [Function],
                "name": "healthcareServices",
                "text": "Healthcare Services",
              },
            ],
            "containerId": "golden-home",
            "stateMetadata": Object {
              "content": Array [
                Object {
                  "content": Array [
                    Object {
                      "content": Array [
                        Object {
                          "activeItemIndex": 0,
                          "content": Array [
                            Object {
                              "componentName": "organizations",
                              "isClosable": true,
                              "reorderEnabled": true,
                              "title": "Organizations",
                              "type": "component",
                            },
                          ],
                          "header": Object {},
                          "height": 50,
                          "isClosable": true,
                          "reorderEnabled": true,
                          "title": "",
                          "type": "stack",
                        },
                        Object {
                          "activeItemIndex": 0,
                          "content": Array [
                            Object {
                              "componentName": "practitioners",
                              "isClosable": true,
                              "reorderEnabled": true,
                              "title": "Practitioners",
                              "type": "component",
                            },
                          ],
                          "header": Object {},
                          "height": 50,
                          "isClosable": true,
                          "reorderEnabled": true,
                          "title": "",
                          "type": "stack",
                          "width": 50,
                        },
                      ],
                      "isClosable": true,
                      "reorderEnabled": true,
                      "title": "",
                      "type": "column",
                      "width": 50,
                    },
                    Object {
                      "content": Array [
                        Object {
                          "activeItemIndex": 0,
                          "content": Array [
                            Object {
                              "componentName": "locations",
                              "isClosable": true,
                              "reorderEnabled": true,
                              "title": "Location",
                              "type": "component",
                            },
                          ],
                          "header": Object {},
                          "height": 50,
                          "isClosable": true,
                          "reorderEnabled": true,
                          "title": "",
                          "type": "stack",
                        },
                        Object {
                          "activeItemIndex": 0,
                          "content": Array [
                            Object {
                              "componentName": "healthcareServices",
                              "isClosable": true,
                              "reorderEnabled": true,
                              "title": "Healthcare Services",
                              "type": "component",
                            },
                          ],
                          "height": 50,
                          "isClosable": true,
                          "reorderEnabled": true,
                          "title": "",
                          "type": "stack",
                          "width": 50,
                        },
                      ],
                      "isClosable": true,
                      "reorderEnabled": true,
                      "title": "",
                      "type": "column",
                      "width": 50,
                    },
                  ],
                  "isClosable": true,
                  "reorderEnabled": true,
                  "title": "",
                  "type": "row",
                },
              ],
              "dimensions": Object {
                "borderGrabWidth": 15,
                "borderWidth": 5,
                "dragProxyHeight": 200,
                "dragProxyWidth": 300,
                "headerHeight": 30,
                "minItemHeight": 10,
                "minItemWidth": 10,
              },
              "isClosable": true,
              "labels": Object {
                "close": "close",
                "maximise": "maximise",
                "minimise": "minimise",
                "popin": "pop in",
                "popout": "open in new window",
                "tabDropdown": "additional tabs",
              },
              "maximisedItemId": null,
              "openPopouts": Array [],
              "reorderEnabled": true,
              "settings": Object {
                "blockedPopoutsThrowError": true,
                "closePopoutsOnUnload": true,
                "constrainDragToContainer": false,
                "hasHeaders": true,
                "popoutWholeStack": false,
                "reorderEnabled": true,
                "reorderOnTabMenuClick": true,
                "responsiveMode": "onload",
                "selectionEnabled": false,
                "showCloseIcon": true,
                "showMaximiseIcon": true,
                "showPopoutIcon": false,
                "tabControlOffset": 10,
                "tabOverlapAllowance": 0,
              },
              "title": "",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
